specifyweb/frontend/js_src/lib/components/FormSliders/RecordSelectorFromIds.tsx

```diff
@@ -218,15 +227,25 @@ export function RecordSelectorFromIds<SCHEMA extends AnySchema>({
                   onClick={(): void => handleRemove('minusButton')}
                 />
               ) : undefined}
-
               {typeof newResource === 'object' ? (
                 <p className="flex-1">{formsText.creatingNewRecord()}</p>
               ) : (
                 <span
                   className={`flex-1 ${dialog === false ? '-ml-2' : '-ml-4'}`}
                 />
               )}
-
+              {hasAttachments && (
+                <Button.Blue onClick={() => setAttachmentState(true)}>
+                  {attachmentsText.attachments()}
```

Comment:
Make this button less prominent. notice that we don't have any buttons with text up in this area as space is precious, and clutter is discouraged. (specify network logo would be replaced by small badges too)

Replace Button.Blue with Button.icon icon="photos"